aries-site/src/examples/templates/FormChildObject/FormChildObjects.js

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import { Box, Button } from 'grommet';
+import { ButtonGroup } from '../ButtonGroup';
+import { FormChildObject } from './FormChildObject';
+
+export const FormChildObjects = ({
+  collection,
+  fields,
+  level,
+  onAdd,
+  onRemove,
+  onRemoveAll,
+  primaryKey,
+  required,
+  summarize,
+  values,
+}) => (
+  <Box>
+    {values?.length
+      ? values.map((obj, index) => (
+          <FormChildObject
+            key={index}
+            collectionName={collection.itemName}
+            index={index}
+            level={level}
```

Comment:
soft opinion but I feel like I forgot what exactly this `level` was as I was going through different files using `FormChildObject` maybe we can change it to `headingLevel`

Reply:
I could see that -- I'll change to `headingLevel`

Reply:
Done.